greenplumpython/experimental/file.py

```diff
@@ -0,0 +1,79 @@
+import base64
+import inspect
+import io
+import pathlib
+import tarfile
+import uuid
+from typing import get_type_hints
+
+import psycopg2
+
+import greenplumpython as gp
+from greenplumpython.func import NormalFunction
+
+_CHUNK_SIZE = 256 * 1024 * 1024  # Must be much < 1 GB
+
+
+@gp.create_function
+def _dump_file_chunk(tmp_archive_name: str, chunk_base64: str) -> int:
+    tmp_archive_base = pathlib.Path("/") / "tmp" / tmp_archive_name
```

Comment:
Fixing the directory to `/tmp` may not be ideal. We probably need to consider give user options to choose.

`XDG_CACHE_HOME` may be preferred as the default. See https://specifications.freedesktop.org/basedir-spec/basedir-spec-latest.html

But this is out of the scope of this PR. If needed, it can be done in a new PR.